velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py

```diff
@@ -58,27 +59,27 @@ def generate_launch_description():
     declare_gps_time = DeclareLaunchArgument(name = 'gps_time', default_value = 'False')
 
     # Args for Pointcloud
-    velodyne_pointcloud_pkg = get_package_share_directory('velodyne_pointcloud')
```

Comment:
Nit: If you agree, can we update the original "Args..." comments to be slightly more clear regarding which node they are for? Something similar to:
```
# Args for velodyne_driver_node
...

# Args for velodyne_convert_node
...
```

Reply:
Done!